include/grpc/grpc.h

```diff
@@ -79,6 +79,9 @@ GRPCAPI void grpc_init(void);
     destroyed. */
 GRPCAPI void grpc_shutdown(void);
 
+/** Returns 1 if the grpc library has been initialized */
+GRPCAPI int grpc_is_initialized(void);
```

Comment:
We'll need a gRFC for this change, as per:

https://github.com/grpc/proposal/blob/master/P3-grfcs-for-core-api-changes.md

Reply:
I would suggest making this API experimental for now and do the gRFC later when experimental tag is removed.

Reply:
Yes, this should have been marked experimental. I added the experimental "tag" to the comment along with a link to the fork support tracking issue regarding stabilizing it in the future.

Reply:
I'm ok with this being experimental as long as its removal/promotion is tracked.

Reply:
Thanks for the review! I kind of messed up my "only review the second commit" plan by then adding a 3rd/4th commit to update this method as experimental. The full comment on this function now includes the tracking issue as well:

```
/** EXPERIMENTAL. Returns 1 if the grpc library has been initialized.
    TODO(ericgribkoff) Decide if this should be promoted to non-experimental as
    part of stabilizing the fork support API, as tracked in
    https://github.com/grpc/grpc/issues/15334 */
```